apps/site/assets/ts/schedule/components/schedule-finder/ScheduleModalContent.tsx

```diff
@@ -130,6 +134,61 @@ const ScheduleModalContent = ({
       ? "All branches"
       : directionDestinations[selectedDirection];
 
+  let ScheduledTripsOrScheduleNote: ReactElement<HTMLElement>;
+
+  if (scheduleNote) {
+    // we will show the schedule note along with the first and
+    // last trip times in lieu of the service picker and trip table.
+    const defaultFirstLastTimes: { [key: string]: [string, string] } = {
+      "Monday - Saturday": ["05:15 AM", "12:30 AM"],
+      Sunday: ["06:00 AM", "12:30 AM"]
```

Comment:
What do you think of locating this data within `schedule_note.ex`, alongside the other static info that gets fed in as the `scheduleNote` property?

Reply:
Yes, that sounds way better. I also just discovered `apps/schedules/lib/hours_of_operation.ex`, which I'm going to see if I can leverage here instead.

Reply:
Indeed, it seems we already have an `HoursOfOperation` React component as well. This is all new stuff to me, glad we're uncovering it 😊